qa/tasks/mgr/dashboard/test_mgr_module.py

```diff
@@ -52,6 +53,7 @@ def test_list_disabled_module(self):
         self.assertIsNotNone(module_info)
         self.assertFalse(module_info['enabled'])
 
+    @retry(on_exception=RuntimeError, tries=2, delay=0.5, logger=log)
     def test_list_enabled_module(self):
         self._ceph_cmd(['mgr', 'module', 'enable', 'iostat'])
         self.wait_until_rest_api_accessible()
```

Comment:
Shouldn't this command be enough to make it work? Maybe the fix needs to happen there?

Reply:
i moved the retry to `wait_until_rest_api_accessible`